src/account/account-data.ts

```diff
@@ -195,4 +194,21 @@ export class AccountData {
       }
     }
   }
+
+  /**
+   * Uploads portable account without registration in ENS
+   *
+   * @param username FDP username
+   * @param password FDP password
+   * @param privateKey account private key for data uploading to SWARM
+   * @param seed account seed to be uploaded in SWARM
+   */
+  async uploadPortableAccount(
+    username: string,
+    password: string,
+    privateKey: PrivateKeyBytes,
+    seed: CryptoJS.lib.WordArray,
+  ): Promise<Reference> {
+    return await uploadPortableAccount(this.connection, username, password, privateKey, seed)
```

Comment:
`privateKey` and `seed` shouldn't be passed, the active account could be used for uploading the credentials since the portable account must be subsumed under the user's IAAS.

This function has to check whether the public key associated with the username in ENS is identical with the wallet's public key, otherwise throw error. Would be nice to call this check function standalone without reuploading the credentials. For that, please provide test as well.

`reuploadPortableAccount` would be better name for this method IMO.